<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>574</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ALGORITMO GENÉTICO CANÓNICO</string>
    </property>
   </widget>
   <widget class="QLabel" name="initial_population_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Población Inicial:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maximum_population_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Población Máxima:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pmg_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>P_mut_gen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pmi_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>P_mut_ind:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pc_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>P_crz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="precision_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Precisión:</string>
    </property>
   </widget>
   <widget class="QLabel" name="initial_value_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Valor inicial:</string>
    </property>
   </widget>
   <widget class="QLabel" name="final_value_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Valor final:</string>
    </property>
   </widget>
   <widget class="QLabel" name="generation_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Cantidad Generaciones:</string>
    </property>
   </widget>
   <widget class="QLabel" name="title2_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Intervalos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="max_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>maximizar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="min_value">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Minimizar</string>
    </property>
   </widget>
   <widget class="QLabel" name="title3_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Valores</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="initial_population_input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>111</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="maximum_population_input">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="pmg_input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="pmi_input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="pc_input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="precision_input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="initial_value_input">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="final_value_input">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="generation_input">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="estado">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
